### ISSUE TITLE
Read csv files for InterpolatedVolumeParameter

### ISSUE THREAD

**Issue by VJander** [pywr/pywr#926]
In a past issue it is made clear that `InterpolatedVolumeParameter` can't read volume and values from external data, and are limited to direct array coding. I need to use a level-volume curve that has over 3000 values and doesn't adjust quite well to many functions, so i need to add the feature of reading external data to this parameter, which files to i have to modify in order to make this pull request? And also, would you like me to upload a `InterpolatedVolumeParameter` test in case of doing it?

**Comment by jetuk** [comment]
Hi @VJander 

I think this would be a good addition. We already have a generic function for load external data: `pywr.parameters._parameters.load_parameter_values`. Therefore, I think you can probably modify the `load` method of `InterpolatedVolumeParameter` https://github.com/pywr/pywr/blob/master/pywr/parameters/parameters.py#L142

Something like this should work:

```python
    @classmethod
    def load(cls, model, data):
        node = model._get_node_from_ref(model, data.pop("node"))

        volumes = data.pop("volumes")
        if isinstance(volumes, list):
            volumes = np.array("volumes")
        elif isinstance(volumes, dict):
            volumes = load_parameter_values(model, volumes)
        else:
            raise TypeError("Unexpected type for \"volumes\" in {}".format(cls.__name__))
        # TODO Repeat above for "values"
        values = np.array(data.pop("values"))
        kind = data.pop("kind", "linear")
        return cls(model, node, volumes, values, interp_kwargs={'kind': kind})
```

You would then be able to specify the JSON like so:

```json
my_interp_volume: {
  "type": "InterpolatedVolume",
  "volumes": {
    "url": "mydata.csv",
    "column": "volumes"
  },
  "values": {
    "url": "mydata.csv",
    "column": "values"
  }
}     
```

A similar approach is used for the `ArrayIndexedScenarioMonthlyFactorsParameter`: https://github.com/pywr/pywr/blob/master/pywr/parameters/_parameters.pyx#L532

**Comment by VJander** [comment]
Thanks James!
Just did the pull request, the different tests didn't check any errors, would you check them out to confirm this?

**Comment by jetuk** [comment]
Closed in #930
